src/App.tsx

```diff
@@ -1,27 +1,86 @@
+import { useState } from 'react';
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoInfo } from './components/TodoInfo/TodoInfo';
```

Comment:
The `TodoInfo` component is imported as a named import, but it should be imported as a default import. Ensure that `TodoInfo` is exported as a default export in its module.